hazelcast/src/main/java/com/hazelcast/scheduledexecutor/impl/operations/SyncStateOperation.java

```diff
@@ -31,7 +32,7 @@
 import static com.hazelcast.util.MapUtil.createHashMap;
 
 public class SyncStateOperation
```

Comment:
Also internal operation, should probably be exempt from quorum checks. Same for backup op.

Reply:
Well, it's a modifying operation, like backup operation, but I removed it, since one can argue that this should proceed in a smaller sub-cluster, even if no quorum, to make sure the nodes are in sync. But these ops are questionable...